bc_obps/registration/fixtures/mock/userOperator.json

```diff
@@ -46,5 +46,281 @@
       "verified_at": null,
       "verified_by": null
     }
+  },
+  {
+    "model": "registration.useroperator",
+    "pk": 5,
+    "fields": {
+      "user": "00000000-0000-0000-0000-000000000002",
+      "operator": 3,
+      "role": "reporter",
```

Comment:
We're not using the reporter role for MVP, so I'd suggest setting everyone to admin so we don't confuse any internal people who are poking around in dev

Reply:
Good catch, I've updated this thanks.